src/common/graphql.ts

```diff
@@ -0,0 +1,17 @@
+import { returnsExpectedStatus } from '../common/util';
+import { subquery } from '../consts';
+
+export async function isGraphQlUp(): Promise<boolean> {
+  const query = `query {
+    blocks(first: 1) {
+      totalCount
```

Comment:
nit -> instead of `totalCount` we should use `{ nodes { id } }`. count operation with limit 1 feels a bit odd